pkg/ir/unspecified.go

```diff
@@ -35,8 +35,22 @@ func (s *Spec) ComputeBlockedSubnets(singleACL bool) {
 				}
 			}
 		}
+		if s.findEndPointInConnections(segments, EndpointTypeSubnet) {
+			continue
+		}
 
-		if !s.findEndPointInConnections(segments, EndpointTypeSubnet) { // EndpointTypeSubnet should be changed to EndpointTypeSegment
+		cidrSements := []string{}                                   // cidr segments which include the subnet
+		for segmentName, cidrSegment := range s.Defs.CidrSegments { // go over cidr segments
+			for _, subnets := range cidrSegment {
+				for _, s := range subnets { // go over the subnets for each cidr
```

Comment:
Avoid comments at the end of a line. Also, the last two comments do not add information.